src/caffe2-ios/caffe2-ios/staticDetectorVC.swift

```diff
@@ -31,17 +33,22 @@ class staticDetectorVC: UIViewController {
         case FoundNil(String)
     }
     
-    @IBAction func runButton(_ sender: UIButton) {
-        self.classifier(name: testImg)
+    @IBAction func demoButton(_ sender: UIButton) {
+        let demoImage = UIImage(named: testImg)!
+        self.classifier(image: demoImage)
+    }
+    
+    @IBAction func pickPhotoFromLibrary(_ sender: UIButton) {
+        self.present(self.imagePickerController, animated: false, completion: nil)
```

Comment:
Once photo uploaded, should the demo button change the text to something like `run` instead of demo? kinda confused then?